src/tests/jsonLogic.test.js

```diff
@@ -0,0 +1,213 @@
+        },
+        a_is_greater_than_c: {
+          errorMessage: 'Field A must be greater than field_c',
+          rule: {
+            '>': [{ var: 'field_a' }, { var: 'field_c' }],
+          },
+        },
+      });
+      const { handleValidation } = createHeadlessForm(schema, { strictInputType: false });
+      expect(handleValidation({ field_a: 1, field_b: 10, field_c: 0 }).formErrors.field_a).toEqual(
+        'Field A must be greater than field_b'
+      );
+      expect(handleValidation({ field_a: 1, field_b: 0, field_c: 10 }).formErrors.field_a).toEqual(
+        'Field A must be greater than field_c'
+      );
+      expect(handleValidation({ field_a: 10, field_b: 5, field_c: 5 }).formErrors).toEqual(
+        undefined
+      );
+    });
+
+    it('OR: field_a > field_b or field_a > field_c', () => {
+      const schema = createSchemaWithThreePropertiesWithRuleOnFieldA({
+        field_a_is_greater_than_b_or_c: {
+          errorMessage: 'Field A must be greater than field_b or field_c',
+          rule: {
+            or: [
+              { '>': [{ var: 'field_a' }, { var: 'field_b' }] },
+              { '>': [{ var: 'field_a' }, { var: 'field_c' }] },
+            ],
+          },
+        },
+      });
+      const { handleValidation } = createHeadlessForm(schema, { strictInputType: false });
+      expect(handleValidation({ field_a: 0, field_b: 10, field_c: 10 }).formErrors.field_a).toEqual(
+        'Field A must be greater than field_b or field_c'
+      );
+      expect(handleValidation({ field_a: 1, field_b: 0, field_c: 10 }).formErrors).toEqual(
+        undefined
+      );
+      expect(handleValidation({ field_a: 10, field_b: 5, field_c: 5 }).formErrors).toEqual(
+        undefined
+      );
+    });
+  });
+
+  describe('Multiple validations', () => {
+    it('2 rules where A must be bigger than B and not an even number in another rule', () => {
+      const { handleValidation } = createHeadlessForm(multiRuleSchema, { strictInputType: false });
+      expect(handleValidation({ field_a: 1 }).formErrors).toEqual({
+        field_a: 'A must be even',
+        field_b: 'Required field',
+      });
+      expect(handleValidation({ field_a: 1, field_b: 2 }).formErrors).toEqual({
+        field_a: 'A must be bigger than B',
+      });
+      expect(handleValidation({ field_a: 3, field_b: 2 }).formErrors).toEqual({
+        field_a: 'A must be even',
+      });
+      expect(handleValidation({ field_a: 4, field_b: 2 }).formErrors).toEqual(undefined);
+    });
+
+    it('2 seperate fields with rules failing', () => {
+      const { handleValidation } = createHeadlessForm(schemaWithTwoRules, {
+        strictInputType: false,
+      });
+      expect(handleValidation({ field_a: 1, field_b: 3 }).formErrors).toEqual({
+        field_a: 'A must be bigger than B',
+        field_b: 'B must be even',
+      });
+      expect(handleValidation({ field_a: 4, field_b: 2 }).formErrors).toEqual(undefined);
+    });
+  });
+});
```

Comment:
Let's take a moment of gratitude/pride for all these tests to be green! 🚀 🥹 Amaaaazing!!

 The code was easy to read, and didn't find any bugs, just tiny comments around wording and descriptions. Thank you a LOT for the effort in breaking this up in smaller PRs.

Reply:
Agree with @sandrina-p 100%! 🙌

Reply:
Yes!!! All those new possibilities are amazing! 🚀🚀🚀

Reply:
thank you all for putting up with me around this effort ❤️